index.js

```diff
@@ -9,8 +9,23 @@
 
 var cp = require('child_process');
 
+var program = {
+	darwin: {
+		name: 'pbcopy',
+		args: []
+	},
+	win32: {
+		name: 'clip',
+		args: [],
+	},
+	linux: {
+		name: 'xclip',
+		args: ['-selection', 'clipboard']
+	}
+};
+
 function toClipboard(args, cb) {
-  var proc = cp.spawn(program(), {
+  var proc = cp.spawn(program.name, program.args, {
```

Comment:
wait, how did this work for you locally? `program.name` doesn't exist. `process.platform` is used to get the value to use. also, you only updated the `sync` code to be a string.